src/common/BaseAnnotator.ts

```diff
@@ -109,15 +108,13 @@ export default class BaseAnnotator extends EventEmitter {
         this.annotatedEl.classList.add(CSS_LOADED_CLASS);
         this.containerEl.classList.add(CSS_CONTAINER_CLASS);
 
-        // Update the store with the options provided by preview
-        this.store.dispatch(store.setRotationAction(rotation));
-        this.store.dispatch(store.setScaleAction(scale));
-
         // Defer to the child class to render annotations
         this.render();
 
         // Update the store now that annotations have been rendered
-        this.store.dispatch(setIsInitialized());
+        this.store.dispatch(store.setRotationAction(rotation));
+        this.store.dispatch(store.setScaleAction(scale));
+        this.store.dispatch(store.setIsInitialized());
```

Comment:
Is there any value in only dispatching this if the annotator is not initialized?